integration-tests/src/test/java/org/apache/druid/tests/security/ITTLSTest.java

```diff
@@ -537,4 +534,19 @@ private StatusResponseHolder makeRequest(
       throw new RuntimeException(e);
     }
   }
+
+  private boolean isRetriable(Throwable ex)
```

Comment:
`isRetriable` seems a little off 😅. Maybe use `canRetry` or `shouldRetry`.

Reply:
seems alright to me 😄